docs/spaces/index.asciidoc

```diff
@@ -92,18 +92,21 @@ image::spaces/images/spaces-roles.png["Controlling features visiblity"]
 [float]
 [[spaces-moving-objects]]
 === Move saved objects between spaces
-Use {kib}'s <<managing-saved-objects-export-objects, import/export>> 
+
+You have two options:
```

Comment:
So you don't have a numbered list inside of a bulleted list, how about something like this:

You can <<managing-saved-objects-copy-to-space, copy objects>> from one space to another 
in *Management > Saved objects.*

Alternately, you can move objects using {kib}'s <<managing-saved-objects-export-objects, import and export>> 
interface.

. Navigate to the space that contains your saved objects.
. Export your saved objects.
. Navigate to the space where you want to import the objects.
. Import your saved objects.
. (Optional) Delete objects in the export space that you no longer need.